datafusion/core/tests/sqllogictests/test_files/window.slt

```diff
@@ -258,34 +258,37 @@ Sort: d.b ASC NULLS LAST
 ----Aggregate: groupBy=[[d.b]], aggr=[[MAX(d.a)]]
 ------SubqueryAlias: d
 --------SubqueryAlias: _data2
-----------SubqueryAlias: s
-------------SubqueryAlias: _sample_data
```

Comment:
> Isn't the new plan worse? Previously there is no `WindowAggr` because `seq is not used in the outer query's select list:
> 
> ```sql
> SELECT d.b, MAX(d.a) AS max_a
> ```

I agree with this observation, also comment in the test (`# window expr is not referenced anywhere, eliminate it.`) suggests that this is the desired behavior and intentional